notes/wcag.md

```diff
@@ -10,6 +10,17 @@ The [Chinese Layout Task Force](https://github.com/w3c/clreq) is also discussing
 
 I will let you know when the group has a consensus on this.
 
+@aphillips notes:
+> 80 characters seems like it might have a historical basis, such as the use of terminals or software
```

Comment:
I agree. We also [discussed](https://www.w3.org/2023/08/08-clreq-minutes.html#t02) this issue in yesterday's clreq TF teleconference, and some people think that the 80 characters limit is very arbitrary, even for English, because the limit is greatly affected by the font size and font.